openwisp-monitoring/files/monitoring.agent

```diff
@@ -172,6 +172,8 @@ send_data() {
 					if [ "$VERBOSE_MODE" -eq "1" ]; then
 						logger -s "Data sent successfully." \
 							-p daemon.info
+						# remove response file if request was failing previously
+						[ "$FAILING" -eq "1" ] && FAILING=0 && rm -f "$RESPONSE_FILE"
```

Comment:
I am not sure I understand this change, why is it necessary?

Reply:
Reverted the changes